templates/theme/styles/base.less

```diff
@@ -1,11 +1,6 @@
 @import "syntax-variables";
 
-.editor-colors {
-  background-color: @syntax-background-color;
-  color: @syntax-text-color;
-}
-
-.editor {
+:host {
```

Comment:
For these selectors, we should retain `atom-text-editor` so they work with the shadow DOM disabled, like this:

```
atom-text-editor, :host {
```